### ISSUE TITLE
Encrypt multiple files and Windows context menu

### ISSUE THREAD

**Issue by slawekdw** [ryangriggs/PDFEncrypt#9]
Helo
Have you thought about the possibility of adding functionality
from the Windows context menu open with or send to and encrypt multiple selected files ?
Regards

**Comment by ryangriggs** [comment]
Yes these features are already in V2 which is not yet released.  Thank you for these suggestions.
